.github/scripts/actionlint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#################################################################
+#    Lint workflows with https://github.com/rhysd/actionlint    #
+#################################################################
+
+# v1.7.7
+readonly ACTIONLINT_VERSION=03d0035246f3e81f36aed592ffb4bebf33a03106
+
+# Verify that shellcheck is installed (preinstalled on GitHub Actions runners)
+if [[ -z "$(command -v shellcheck)" ]]; then
+    error 'This script requires shellcheck to be installed. Please install it and try again'
+    exit 1
+fi
+
+SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+readonly SCRIPT_DIR
+GITHUB_DIR="$(dirname "$SCRIPT_DIR")"
+readonly GITHUB_DIR
+
+source "$SCRIPT_DIR/shellUtils.sh"
+
+cd "$GITHUB_DIR" || exit 1
```

Comment:
How is this script meant to be used - does it only lint the actions in this repo or is it meant to be used in other repos? If other repos, you probably want to pass the github directory in as an argument. If this repo, then you can simplify all this since you don't use GITHUB_DIR anywhere else and there's no good reason for `cd` to fail:
```suggestion
SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
readonly SCRIPT_DIR

source "$SCRIPT_DIR/shellUtils.sh"

cd "$SCRIPT_DIR/.."
```

Reply:
That's actually a good point. I've never built a `pull_request` workflow that's meant to run in multiple different repos. I suggest for now that we address that in a follow up. For now, I'll pretend it's only meant for use in this repo, and simplify it as you suggest.

Also, that pattern that you see with `cd <dir> || exit 1` is just to make shellCheck happy.

Reply:
What's the shellcheck complaint?

Reply:
Actually, I found a great way to simplify this in that the `cd` isn't needed at all. It will work as long as you run it for the repo you're trying to lint.

In order to make this script work in other repos, we'll probably have to make some changes and cd to that other repo after sourcing shellUtils but before running actionlint